aha-table.html

```diff
@@ -429,15 +429,25 @@
       var scrollBody = Polymer.dom(aha.root).querySelector('.scroll-body.aha-table');
       var rowHeight = 42;
 
+      var closestPageSize;
+
       var optimizePageSize = Math.floor((Number(scrollBody.offsetHeight) - Number(th.offsetHeight)) / rowHeight);
       if (optimizePageSize <= 10) {
```

Comment:
this method `_optimizeForPageLoad` should probably be renamed to `_initializeTable`